client/extensions/woocommerce/app/order/order-notes/new-note.js

```diff
@@ -60,11 +59,11 @@ class CreateOrderNote extends Component {
 		return (
 			<div className="order-notes__new-note">
 				<FormFieldSet className="order-notes__new-note-content">
-					<FormLabel htmlFor="note">{ translate( 'Add a note' ) }</FormLabel>
```

Comment:
Derp, there I was looking at the FormLabel component for a screen reader property.

I'll fix.